optaplanner-integration/optaplanner-spring-boot-example/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+# The solver runs for 30 seconds. To run for 5 minutes use "5M" and for 2 hours use "2H".
+optaplanner.solver.termination.spent-limit=30S
```

Comment:
We're showing users how to do it. Let's not confuse them and let them believe they should use capital letters for time units. While ISO 8601 is one of the formats supported by Spring Boot, I'd show them the more natural way of denoting duration (`30s`) and also show them they can combine units: `2h45m`.